src/components/AMMPrices/TokenAMMPriceRow.tsx

```diff
@@ -56,6 +56,14 @@ export default function TokenAMMPriceRow({
     )
   }
 
+  const formatPrice = (price: string) => {
+    const p = parseInt(price, 10)
+    const formatLimit = 1e12 - 1 // trillion
```

Comment:
**Suggestion: ** add `const` for trillion in `constants/numbers` and use it here :)

Reply:
Thank you for this suggestion ser, i felt a bit dirty leaving it here :/